src/Tools/ExternalAccess/Format/Microsoft.CodeAnalysis.ExternalAccess.Format.csproj

```diff
@@ -0,0 +1,39 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <RootNamespace>Microsoft.CodeAnalysis.ExternalAccess.Format</RootNamespace>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
not netstandard?

Reply:
Since there will only ever be one consumer for this library, I'm not sure it matters.